install.sh

```diff
@@ -53,6 +53,44 @@ printSuccess() {
   echo "${green}${msg[*]}${reset}"
 }
 
+checkJwt() {
+  local is_sandbox=$1
+  local php_path=$2
+  local environment="production"
+  if [[ $is_sandbox ]]; then environment="sandbox"; fi
+  if [[ -f pub/index.php ]]
```

Comment:
It's not guaranteed this file exists. It could be in application root or some multi websites might use something more sophisticated pub/en_US/index.php

Validate agains env.php as https://github.com/magento-swat/install-agent-helpers/pull/21/files#diff-043df5bdbf6639d7a77e1d44c5226fd7371e5259a1e4df3a0dd5d64c30dca44fR207